server/commands/hubs.py

```diff
@@ -575,11 +576,41 @@ def ooc_cmd_unhide_clients(client, arg):
     )
 
 
+@mod_only(hub_owners=True)
+def ooc_cmd_follow_me(client, arg):
```

Comment:
You should probably rename /follow_me to something like /force_follow so it's more obvious it's a follow you are forced into.